lib/metrics/gc.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const Counter = require('../counter');
+const Histogram = require('../histogram');
+
+let perf_hooks;
+
+try {
+	// eslint-disable-next-line
+	perf_hooks = require('perf_hooks');
+} catch (e) {
+	// node version is too old
+}
+
+const NODEJS_GC_RUNS_TOTAL = 'nodejs_gc_runs_total';
```

Comment:
Naming looks good. I've the impression that prometheus discourages duplicate collection of metrics, and the totals are already available in the duration counts below, but I'm not expert enough to know for sure.

Reply:
metric removed